### PR TITLE
operators: add cluster-openshift-apiserver-operator

### PR DESCRIPTION
**Pull request by juanvallejo**
Not sure if there's anything to generate.
Is there an operator registry somewhere / any files I may have missed when adding a new operator?

cc @deads2k @mfojtik @yifan-gu @abhinavdahiya

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
Requires changes from https://github.com/coreos-inc/tectonic-operators/issues/464 so that we can actually check if we can bootstrap using this operator

File: modules/tectonic/resources/manifests/updater/operators/cluster-openshift-apiserver-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+apiVersion: v1
```
[the remark this answers is not in the document]
Comment (juanvallejo) [reply modules/tectonic/resources/manifests/updater/operators/cluster-openshift-apiserver-operator.yaml]: Sure, will update

**Comment by abhinavdahiya** [comment]
Also https://github.com/coreos-inc/tectonic-operators/pull/464#issuecomment-423590240

It might be private
@deads2k 
>  apiservices are created for the openshift-apiserver by the cluster-openshift-apiserver-operator. Since the openshift-apiserver isn't required to have a viable control plane, there is no reason for it have any static resources created, so the operator can manage all of it.

> @juanvallejo you will want to be sure that the service-ca operator is installed and active in the installer though.

**Comment by juanvallejo** [comment]
@abhinavdahiya Opened PR to add the openshift-service-ca-operator here: https://github.com/openshift/installer/pull/306

**Comment by abhinavdahiya** [comment]
/close

The operator is installed by CVO; https://github.com/openshift/installer/pull/430 enables the operator.

**Pull request closed without merging**